src/Http/Livewire/DeleteUserForm.php

```diff
@@ -16,6 +18,27 @@ class DeleteUserForm extends Component
      */
     public $confirmingUserDeletion = false;
 
+    /**
+     * The user's current password.
+     *
+     * @var string
+     */
+    public $password = '';
+
+    /**
+     * Confirm that the user would like to logout from other browser sessions.
```

Comment:
Confirm that the user would like to **delete their account**.

Reply:
Good pickup, resolved.